File: despatch.WEB/app/src/swagger/models/contact-data-organizations-request.ts
/* tslint:disable */
import { BaseEntityRequest } from './base-entity-request';
import { ContactDataRequest } from './contact-data-request';
import { OrganizationRequest } from './organization-request';
export interface ContactDataOrganizationsRequest extends BaseEntityRequest{
  contactData?: ContactDataRequest;
  contactDataId?: string;
  organization?: OrganizationRequest;
  organizationId?: string;
}

File: despatch.Domain/Models/Default/Request/ContactDataOrganizationsRequest.cs
namespace despatch.Domain.Models.Default.Request
{
    public class ContactDataOrganizationsRequest : BaseEntityRequest
    {
        public string ContactDataId { get; set; }
        public string OrganizationId { get; set; }

        public virtual ContactDataRequest ContactData { get; set; }
        public virtual OrganizationRequest Organization { get; set; }
    }
}
